File: sdk/js/examples/01_node_quickstart/is_prime_test.ts
// Copyright 2021 Touca, Inc. Subject to Apache-2.0 License.

import { touca } from '@touca/node';
import { is_prime } from './is_prime';

touca.workflow('is_prime_test', (testcase: string) => {
  const number = Number.parseInt(testcase);
  touca.add_result('is_prime_output', is_prime(number));
});

touca.run();

File: sdk/python/examples/01_python_minimal/is_prime_test.py
# Copyright 2021 Touca, Inc. Subject to Apache-2.0 License.

import touca


@touca.Workflow
def is_prime(testcase: str):
    touca.add_result("is_prime", is_prime(int(testcase)))


if __name__ == "__main__":
    touca.run()
